### PR TITLE
Attachments model added

### PR DESCRIPTION
**Pull request by Winchestery**
Added an attachments model
image

### CODE REVIEW

File: src/models/attachments/reducers/attachment-current.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    case types.FETCHING_ATTACHMENT_BY_ID_FAILED:
+      return {
+        ...state,
+        isLoading: false,
+        isLoaded: false,
+        error: action.payload.error.message,
+      };
+
+    case types.CLEAR_LOADED:
```
Comment (MozdolevskyiMaksym) [inline src/models/attachments/reducers/attachment-current.js:36]: Why do you use this type here?

File: src/models/attachments/reducers/attachment-current.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      return {
+        ...state,
+        loaded: false,
```
Comment (MozdolevskyiMaksym) [inline src/models/attachments/reducers/attachment-current.js:39]: `isLoaded`

File: src/models/attachments/reducers/attachments-all-add.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        ...state,
+        isLoading: false,
+        isLoaded: false,
+        error: action.payload.error.message,
+      };
+
+    case types.CLEAR_LOADED:
+      return {
+        ...state,
+        loaded: false,
```
Comment (MozdolevskyiMaksym) [inline src/models/attachments/reducers/attachments-all-add.js:39]: `isLoaded`

File: src/models/attachments/reducers/attachments-all.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+    case types.FETCHING_ATTACHMENTS_FAILED:
+      return {
+        ...state,
+        isLoading: false,
+        isLoaded: false,
+        error: action.payload.error.message,
+      };
+
+    case types.CLEAR_LOADED:
```
Comment (MozdolevskyiMaksym) [inline src/models/attachments/reducers/attachments-all.js:36]: I think we don`t need this type here

File: src/models/attachments/types.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+export const FETCHING_ATTACHMENTS_STARTED = 'ATTACHMENTS/FETCHING_ATTACHMENTS_STARTED';
+export const FETCHING_ATTACHMENTS_SUCCEED = 'ATTACHMENTS/FETCHING_ATTACHMENTS_SUCCEED';
+export const FETCHING_ATTACHMENTS_FAILED = 'ATTACHMENTS/FETCHING_ATTACHMENTS_FAILED';
+
+export const FETCH_ATTACHMENT_BY_ID = 'ATTACHMENTS/FETCH_ATTACHMENT_BY_ID';
+export const FETCHING_ATTACHMENT_BY_ID_STARTED = 'ATTACHMENTS/FETCHING_ATTACHMENT_BY_ID_STARTED';
+export const FETCHING_ATTACHMENT_BY_ID_SUCCEED = 'ATTACHMENTS/FETCHING_ATTACHMENT_BY_ID_SUCCEED';
+export const FETCHING_ATTACHMENT_BY_ID_FAILED = 'ATTACHMENTS/FETCHING_ATTACHMENT_BY_ID_FAILED';
+
+export const CLEAR_LOADED = 'ATTACHMENT/CLEAR_LOADED';
```
Comment (MozdolevskyiMaksym) [inline src/models/attachments/types.js:16]: `attachmentS`

Comment (Winchestery) [reply src/models/attachments/types.js:16]: done

File: src/models/attachments/reducers/attachments-all.js (hunk 4 above)
Comment (Winchestery) [reply src/models/attachments/reducers/attachments-all.js:36]: done

File: src/models/attachments/reducers/attachments-all-add.js (hunk 3 above)
Comment (Winchestery) [reply src/models/attachments/reducers/attachments-all-add.js:39]: done

File: src/models/attachments/reducers/attachment-current.js (hunk 2 above)
Comment (Winchestery) [reply src/models/attachments/reducers/attachment-current.js:39]: done

File: src/models/attachments/reducers/attachment-current.js (hunk 1 above)
Comment (Winchestery) [reply src/models/attachments/reducers/attachment-current.js:36]: ok, i have changed it

**Review by MozdolevskyiMaksym — APPROVED** [review]

**Review by so2niko — APPROVED** [review]

**Review by MarinaPasternak — APPROVED** [review]

**Review by sarhan-azizov — APPROVED** [review]

**Pull request merged**